src/tools/lifter/bir_inst_liftingLib.sml

```diff
@@ -1694,6 +1694,11 @@ structure bir_inst_liftingLib :> bir_inst_liftingLib = struct
   structure bmil_m0_BigEnd_Process    = bir_inst_liftingFunctor (struct val mr = m0_bmr_rec_BigEnd_Process end);
   structure bmil_m0_BigEnd_Main       = bir_inst_liftingFunctor (struct val mr = m0_bmr_rec_BigEnd_Main end);
 
+  structure bmil_m0_mod_LittleEnd_Process = bir_inst_liftingFunctor (struct val mr = m0_mod_bmr_rec_LittleEnd_Process end);
+  structure bmil_m0_mod_LittleEnd_Main    = bir_inst_liftingFunctor (struct val mr = m0_mod_bmr_rec_LittleEnd_Main end);
+  structure bmil_m0_mod_BigEnd_Process    = bir_inst_liftingFunctor (struct val mr = m0_mod_bmr_rec_BigEnd_Process end);
+  structure bmil_m0_mod_BigEnd_Main       = bir_inst_liftingFunctor (struct val mr = m0_mod_bmr_rec_BigEnd_Main end);
```

Comment:
I second the importance of limits to line length.

Reply:
ok, you're right. i will make them shorter from now on. but 100 is the smallest of what i can accept as a limit :). is this possible to fit on your split screen? i can do slightly more than 120 on my setup.

with your arguments in mind it is funny enough that many people don't apply the same reasoning to sentence length. oh well

Reply:
We should have some kind of vote on this, I think. Hell, I think we should have a meeting to decide on code style in general.

I prefer something like 80, since I think this increases legibility. This is an opinion that was formed a very long time ago however, I haven't really challenged that view for some time now.

Maybe I'm getting too autistic to understand your joke here, but the analogy with sentences is not correct. We are talking about indentation and line breaks for a sentence, not (typically) about shortening or lengthening the sentence itself. Also, people usually do apply the same reasoning to regular text. See for example: [this](http://webtypography.net/2.1.2) and [this](https://baymard.com/blog/line-length-readability) page.

Reply:
I :+1: the "meeting to decide on code style in general".

Reply:
I think a meeting at this point would not be well spent time. But we can see whether we can setup some suggested guidelines of what we feel would be practical for the work.

Instead of starting with the meeting, I suggest that Thomas L sets up a good example. Thomas could indicate one existing little Lib file which follows something he likes and possibly making it more consistent. Alternatively, he could setup a new little mini tool, which does nothing useful but is just an example. This could sit in src/examples and be just a good example for readable code to be guided by.

Ultimately, it would be nice to have an examples for usage in src/examples. This could explain the usage of HolBA by example and we could call this howto-holba or howto-code or something else. It could be worked out to reflect our most common ideas on readable code and at the same time a template of some sort. In this way, we know what we are talking about when it comes to readability.

Reply:
Let's stop discussing this here. I opened #29 for this only.